docs/program/apis/data-client.md

```diff
@@ -178,6 +187,356 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- tags (List[str]): List of tags to remove from specified binary data.
+- filter ([viam.proto.app.data.Filter](https://python.viam.dev/autoapi/viam/proto/app/data/index.html#viam.proto.app.data.Filter "viam.proto.app.data.Filter")): Filter specifying binary data to untag. If no Filter is provided, all data will be
+  untagged.
+
+**Returns:**
+
+- (`int`): The number of tags removed.
+
+**Raises:**
+
+- GRPCError – If no tags are provided.
+
+```python {class="line-numbers linkable-line-numbers"}
+from viam.proto.app.data import Filter
+
+my_filter = Filter(component_name="my_camera")
+tags = ["tag1", "tag2"]
+res = await data_client.remove_tags_from_binary_data_by_filter(tags, my_filter)
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/data_client/index.html#viam.app.data_client.DataClient.remove_tags_from_binary_data_by_filter).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### TagsByFilter
+
+Get a list of tags using a filter.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- filter ([viam.proto.app.data.Filter](https://python.viam.dev/autoapi/viam/proto/app/data/index.html#viam.proto.app.data.Filter "viam.proto.app.data.Filter")): Filter specifying data to retrieve from. If no Filter is provided, all data tags will
+  return.
+
+**Returns:**
+
+- (`List[str]`): The list of tags.
+
+```python {class="line-numbers linkable-line-numbers"}
+from viam.proto.app.data import Filter
+
+my_filter = Filter(component_name="my_camera")
+tags = await data_client.tags_by_filter(my_filter)
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/data_client/index.html#viam.app.data_client.DataClient.tags_by_filter).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### BoundingBoxLabelsByFilter
+
+Get a list of bounding box labels using a Filter.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- filter ([viam.proto.app.data.Filter](https://python.viam.dev/autoapi/viam/proto/app/data/index.html#viam.proto.app.data.Filter "viam.proto.app.data.Filter")): Filter specifying data to retrieve from. If no Filter is provided, all labels will
+  return.
+
+**Returns:**
+
+- (`List[str]`): The list of bounding box labels.
+
+```python {class="line-numbers linkable-line-numbers"}
+from viam.proto.app.data import Filter
+
+my_filter = Filter(component_name="my_camera")
+bounding_box_labels = await data_client.bounding_box_labels_by_filter(
+    my_filter)
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/data_client/index.html#viam.app.data_client.DataClient.bounding_box_labels_by_filter).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GetDatabaseConnection
+
+Get a connection to access a MongoDB Atlas Data federation instance.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- organization_id (str): Organization to retrieve the connection for.
```

Comment:
Needs type link-- also how do you get organization ID? I think it's the CLI and that's kind of hard to find so worth mentioning here